internal/proto/local/controller/storage/host/plugin/store/v1/host.proto

```diff
@@ -93,6 +93,10 @@ message HostSet {
   // preferred_endpoints stores string preference values
   // @inject_tag: `gorm:"-"`
   repeated string preferred_endpoints = 11;
+
+  // Sync interval is a value representing a duration in seconds
+  // @inject_tag: `gorm:"default:null"`
+  int32 sync_interval_seconds = 12 [(custom_options.v1.mask_mapping) = {this: "sync_interval_seconds" that: "sync_interval_seconds"}];
```

Comment:
I think the repo expects casing like `SyncIntervalSeconds` in the mask.

Here and in static.proto

Reply:
When I was looking in the protos most of the mask mapping values did not have such casing, e.g. higher up in the same file.

It's not clear to me if it matters, although I can't test update yet for obvious reasons.

Reply:
Updated